next-app/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
```

Comment:
need to update the vanilla README to mention that it was bazel-ified, maybe point to the separate commit that does it?